templates/configmaps_st2-conf.yaml

```diff
@@ -48,4 +47,10 @@ data:
   # User-defined st2 config with custom settings applied on top of everything else.
   # The order of merging: st2.conf < st2.docker.conf < st2.user.conf
   st2.user.conf: |
+  {{- if .Values.st2.rbac.enabled }}
+    [rbac]
+    sync_remote_groups = True
+  {{- end }}
+
```

Comment:
Can we move this part to the previous `st2.docker.conf` section a few lines before?

Additionally, looks like the configuration to enable RBAC needs to be up to date with the https://docs.stackstorm.com/latest/rbac.html#enabling-rbac instructions